src/components/InfinityLoader.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+<div class="spiner-container">
+  <div class="loadingio-spinner-interwind-2rcyvyzhpsp"><div class="ldio-3igtohkgnvp">
+    <div><div><div><div></div></div></div><div><div><div></div></div></div></div>
+    </div></div>
+</div>
+</template>
+
+<script>
+export default {
+props: ['options'],
+  data: () => ({
+    observer: null,
+  }),
+  mounted() {
+    const options = this.options || {};
+    this.observer = new IntersectionObserver(([entry]) => {
+      if (entry && entry.isIntersecting) {
+        this.$emit('load');
+      }
+    }, options);
+
+    this.observer.observe(this.$el);
+  },
+  destroyed() {
+    this.observer.disconnect();
+  },
+};
+</script>
+
+<style>
+@keyframes ldio-3igtohkgnvp-r {
```

Comment:
`3igtohkgnvp` - ?
You can clean up your code when copy, and use scoped styles to isolate these classes within current component